lib/arn_test.go

```diff
@@ -77,3 +77,8 @@ func testParseARNFail(t *testing.T, arn string) {
 		return
 	}
 }
+
+type roleTestCases struct {
+	expectedOutcome bool
+	testArn         string
+}
```

Comment:
Looks like this was left over from a previous patchset. We should remove it before we merge this pull request. 😃